app/directors-corner.tsx

```diff
@@ -1,25 +1,31 @@
 import Image from 'next/image';
 import Link from 'next/link';
-
 import HorsesRunning from '@/components/horses-running';
 
 export default function DirectorsCorner() {
   return (
     <article className="container my-10 min-w-full">
       <header className="mx-auto mb-20 flex max-w-fit flex-row text-primary-20">
-        <HorsesRunning direction="left" />
-        <h2 className="my-auto">DIRECT</h2>
-        <Image
-          alt="O"
-          className="my-auto h-[44px] rotate-[90.66deg]"
-          height={44}
-          width={44}
-          src="https://s3-alpha-sig.figma.com/img/d003/e4ca/46983bb9c2b53ca4f74a6382f4bfd57d?Expires=1707696000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=PaN9iN-Wh5U0K4D3p08lFy2vBXAuRS7835uwTzaGDtTKWYL0PBZ0Gw4yVjKtcbQMHiscxyh03PgyA4EyTRS0~sXpO14m1RJT8ch27VGGr6fa9zBXWpttPuHgX52tI9S-sodxlmj20HnYYxNObqIDxEkvzgCFJiHsUjVwmxQfOWxwduL80aBRKVUa6bGGSm2V2A2Znl96nJ1yUDdxenBvrTNN4UCFiIJEwyvdr4jVDoWxoqfDveVUcYElzjBjTTYvphifTSeLjEn-uHbX8JmmW1HkqEpThkWsDUVkoGPk~vLBP8LY6fW3aoa7S1ionsMVjG09UsnDCfB7T4Up40LZ6g__"
-        />
-        <h2 className="my-auto">R&rsquo;S CORNER</h2>
+        <div className="hidden md:block">
```

Comment:
divs aren't allowed in the repo. I'd say edit the horses running component to allow taking in breakpoints